conda_recipe_manager/commands/bump_recipe.py

```diff
@@ -0,0 +1,59 @@
+"""
+:Description: CLI for bumping build number in recipe files.
+"""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from typing import cast
+
+import click
+
+from conda_recipe_manager.commands.utils.print import print_err
+from conda_recipe_manager.commands.utils.types import ExitCode
+from conda_recipe_manager.parser.recipe_parser import RecipeParser
+from conda_recipe_manager.types import JsonPatchType
+
+
+@click.command(short_help="Bump recipe build version.")
+@click.argument("recipe_file_path", type=click.Path(exists=True, path_type=str))
```

Comment:
We should add a note about why this isn't a `Path` object, like we did in the `patch` command.